src/background/popupHandler.ts

```diff
@@ -12,7 +12,7 @@ const popups: Dictionary = {};
 
 export const getAeppUrl = (v: any) => new URL(v.connection.port.sender.url);
 
-export const showPopup = async (aepp: any, type: string, params?: any) => {
+export const openPopup = async (type: PopupType, aepp: any, params?: any) => {
```

Comment:
I changed the name of this action to match the `openModal`. So now we have:
```
openPopup
openModal
```

Reply:
Nice.